app.js

```diff
@@ -816,6 +818,51 @@ define(function(require) {
 			});
 		},
 
+		createUserCallflow: function(data, callback, callbackError) {
+			var self = this,
+				resultData = {};
+
+			monster.waterfall([
+				function(waterfallCallback) {
+					self.createUser(data.user,
+						function(userData) {
+							var userId = userData.data.id;
+							data.user.id = userId;
```

Comment:
seems userId is used just here, so, in can be assigned direcly to `data.user.id = userData.data.id`.

Also, I wonder if it will be a good idea to add a lodash#get guard for it just to prevent `userData.data` to be undefined

like
```suggestion
							data.user.id = _.get(userData, 'data.id', '');
```

Reply:
Thanks! :shipit: